hawc/apps/animal/models.py

```diff
@@ -1242,7 +1242,7 @@ def percentControl(data_type, egs):
                 if i_0 is not None and i_0 > 0:
                     i_1 = eg["incidence"]
                     n_1 = eg["n"]
-                    mean = ((i_1 / n_1) - (i_0 / n_0)) / (i_0 / n_0)
+                    mean = ((i_1 / n_1) - (i_0 / n_0)) / (i_0 / n_0) * 100
```

Comment:
looks correct! you can see we do the same on line 1219 `  mean = (mu_2 - mu_1) / mu_1 * 100.0`